docs/proposals/resource-catalog.md

```diff
@@ -0,0 +1,222 @@
+
+
+We can reshape the catalog resource to more easily express them, secure them, and claim them.
+
+# Goals
+ 1.  cluster-admin should control which catalog exist.
+ 2.  cluster-admin should be able/disable per-catalog exposure on a per-project granularity.
```

Comment:
> Why not role based, i.e. is per project more or less valuable than role?

I think you'd want to prevent some projects from exposing resources even if a user has authority to do it.